cammac60/MacRae-Photo-BE
# Post

### What does this PR do? 

- Adds a post endpoint to post messages.
- Adds tests for failing and passing posts. 

----------

### How to manually test: 

- `git fetch`
- `git checkout post`
- `npm test` to ensure that tests are passing. 
- `npm start`
- Make a POST request to `localhost:3000/api/v1/messages` with the proper body format. Ensure that a response ID is returned. 
- Make another POST to the url with no email property on the body. Ensure that you receive the correct error. 

---------

### Tickets: 

- Resolves #4 
- Resolves #3
